tools/run_tests/build_artifact_python.sh

```diff
@@ -81,5 +81,19 @@ then
   done
 fi
 
+# We need to use the built grpcio-tools/grpcio to compile the health proto
+# Wheels are not supported by setup_requires/dependency_links, so we
```

Comment:
Do you mean that `setup_requires` doesn't retrieve wheels or that our wheels in our directory aren't retrieved by `setup_requires` and there's no way to configure that to happen?